src/spec_tools/transformer.cljc

```diff
@@ -61,7 +68,15 @@
     (try
       #?(:clj  (.parse (StdDateFormat.) x)
          :cljs (js/Date. (.getTime (goog.date.UtcDateTime.fromIsoString x))))
-      (catch #?(:clj  Exception, :cljs js/Error) _ x))
+      (catch #?(:clj Exception, :cljs js/Error) _ x))
+    x))
+
+(defn date->string [_ x]
```

Comment:
I think this function and `keyword->string` above are missing tests. Also, it would be very nice to have tests that do the roundtrip, that would help showing/uncover cases in which the "bijection" does not hold.